spec/models/apartments_spec.rb

```diff
@@ -1,7 +1,51 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
-  it("Apartment model exists") do
-  expect{Apartment.create}.to_not raise_error 
+
+  it 'is not valid without a street' do
+    Sza = Apartment.create street: '', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs.', bedrooms: '7', bathrooms: '7', image: "N/A", user_id: 'SzaSeason' 
+    # User = Users.create email: 'szaseason@yahoo.com', encrypted_password: 'HotGirlSummer305'
```

Comment:
You will need to create a user for each of these tests as you must have a user_id associated to something.  Devise will auto reject otherwise and will give you a false positive on these tests.  The README for the application has a helpful line of code for setting this up, and make sure each apartment is associated to that user.